common/persistence/cassandra/cassandraPersistence.go

```diff
@@ -2284,54 +2292,51 @@ func (d *cassandraPersistence) LeaseTaskQueue(request *p.LeaseTaskQueueRequest)
 func (d *cassandraPersistence) UpdateTaskQueue(request *p.UpdateTaskQueueRequest) (*p.UpdateTaskQueueResponse, error) {
 	tli := *request.TaskQueueInfo
 	tli.LastUpdateTime = timestamp.TimeNowPtrUtc()
-	if tli.Kind == enumspb.TASK_QUEUE_KIND_STICKY { // if task_queue is sticky, then update with TTL
-		expiry := types.TimestampNow()
-		expiry.Seconds += int64(stickyTaskQueueTTL)
-
-		datablob, err := serialization.TaskQueueInfoToBlob(&tli)
-		if err != nil {
-			return nil, convertCommonErrors("UpdateTaskQueue", err)
-		}
+	datablob, err := serialization.TaskQueueInfoToBlob(&tli)
+	if err != nil {
+		return nil, convertCommonErrors("UpdateTaskQueue", err)
+	}
 
-		query := d.session.Query(templateUpdateTaskQueueQueryWithTTL,
+	var applied bool
+	previous := make(map[string]interface{})
+	if tli.Kind == enumspb.TASK_QUEUE_KIND_STICKY { // if task_queue is sticky, then update with TTL
```

Comment:
Can you make sure we have unit tests for both code paths `sticky`  and `regular`?

Reply:
we have, however the unit test was asserting the wrong behavior:
https://github.com/temporalio/temporal/pull/935/files#diff-6349d51f67c5e6f3d3ea5fb9bf919a04a5856a06738d80ef17e7a89240709a95L384